app/services/rural-payments-portal.js

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios'
```

Comment:
Is this frontend code? there is no reason for all these dependencies

Reply:
Oh really? Can you show me how i can reduce them?

Reply:
Please use node's built-in library `https` with API `https.request`. axios mainly used for frontend and since fetch is introduced; no new code uses this library

Reply:
As discussed, we are going to fix forward and use the @apollo/datasource-rest. The @apollo/datasource-rest library uses a third party library in any case so the point of using the native http/https lib is mute in any case